### DISCUSSION TITLE
how can I hide/disable the huge ?hints? in rust.

### DISCUSSION THREAD

**Discussion by j-piccolo** [helix-editor/helix#3586] [Q&A] [answered]
<img width="906" alt="image" src="https://user-images.githubusercontent.com/82730686/187067172-9655e9f9-9b0a-4c29-ac69-11d5b60aea2c.png">
I'd like to hide/disable the preview of tokio::spawn when writing within the closure.

**Comment by the-mikedavis** [comment, marked as answer]
You can disable signature help entirely or just the documentation preview with config: https://docs.helix-editor.com/master/configuration.html#editorlsp-section

Recently rust-analyzer was patched to not show signature help on multi-line expressions though: [rust-lang/rust-analyzer@`4bb5df0` (#11791)](https://github.com/rust-lang/rust-analyzer/pull/11791/commits/4bb5df0ce54544df5cf89a08067f7b295b242020) which is in rust-analyzer 2022-03-28 or newer

**Reply by j-piccolo** [reply]
Thanks for that. Weird though, because I tried version 2022-08-22 to start and then nightly.
